components/src/core/Drawer/Drawer.tsx

```diff
@@ -215,6 +218,15 @@ export const DrawerComponent: React.FC<DrawerComponentProps> = (props) => {
     const containerWidth = isDrawerOpen() ? width || defaultContentWidth : defaultNavigationRailWidth;
     const contentWidth = width || defaultContentWidth;
 
+    // TODO: Find a better way to rerender.
+   // Initial draw doesn't have parent container rendered so margin not applied on first load.
+    setTimeout(() => {
+      const content = document.getElementById('@@pxb-drawerlayout-content');
+      if (content) {
+         content.style.marginLeft = `${containerWidth}px`;
+      }
+    });
```

Comment:
- might this be better as a useEffect hook to only run when the open property changes?
- would this logic make more sense in the DrawerLayout...presumably, they may be using the Drawer component without one

Reply:
I agree it makes mores sense to move this logic to the DrawerLayout component, but it makes the complexity a bit harder.  There's one situation where I'm having trouble getting the refresh logic to work.  Whenever the Drawer NavItems are hovered, the Drawer expands.  When this happens, the DrawerLayout now needs to know that it a redraw is required.  Not sure on how to do this functionality.

Reply:
I kept the redraw logic in the Drawer and used useEffect.  This works.

Reply:
The attempted redrawer logic refactor from before was put in a branch called feature/fix-drawerlout-relocated-redraw